### PR TITLE
added CodeSee clickable to the README.md

### LINKED ISSUE #798: Request to add a CodeSee.io Codebase Map
Hey @dwertent, this is a suggestion. How about adding a [CodeSee](https://www.codesee.io/) map for your codebase. It becomes way more accessible for new contributors to work and contribute to the repo.

I have an auto-generated example [here](https://app.codesee.io/maps/public/ea279b60-31d6-11ed-ae63-7d65ece0c3cb) that I was able to produce from my Kubescape fork. Just remember to switch it from **_View Mode_** to **_Playground Mode_** to view the entire file structure.

I believe this will also be helpful for future Onboarding purposes too.

Btw, I love what you guys have made, and have been actively trying to learn the Codebase. And hopefully, make a meaningful contribution.

### PR DESCRIPTION
**Pull request by RayAsh37**
## Describe your changes
I have added CodeSee architecture diagram Clickable in the README.md

## Screenshots - If Any (Optional)
CodeSee

## This PR fixes:
This PR is related to the issue "Request to add a CodeSee.io Codebase Map" 

* Resolved #798  

## Checklist before requesting a review

- [x] My code follows the style guidelines of this project
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] I have performed a self-review of my code
- [x] If it is a core feature, I have added thorough tests.
- [x] New and existing unit tests pass locally with my changes

**Please open the PR against the `dev` branch (Unless the PR contains only documentation changes)**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,6 +7,7 @@
 [![build](https://github.com/kubescape/kubescape/actions/workflows/build.yaml/badge.svg)](https://github.com/kubescape/kubescape/actions/workflows/build.yaml)
 [![Go Report Card](https://goreportcard.com/badge/github.com/kubescape/kubescape)](https://goreportcard.com/report/github.com/kubescape/kubescape)
 [![Gitpod Ready-to-Code](https://img.shields.io/badge/Gitpod-Ready--to--Code-blue?logo=gitpod)](https://gitpod.io/#https://github.com/kubescape/kubescape)
+[![View Code Diagram badge for CodeSee](https://codesee-docs.s3.amazonaws.com/badge.svg)](https://app.codesee.io/maps/public/ea279b60-31d6-11ed-ae63-7d65ece0c3cb)
```
Comment (dwertent) [inline README.md:10]: `View Code Diagram badge for CodeSee` -> `Code Diagram`

**Review by dwertent — CHANGES_REQUESTED** [review]
Great idea!

File: README.md (hunk 1 above)
Comment (RayAsh37) [reply README.md]: @dwertent done

**Comment by RayAsh37** [comment]
Hi @dwertent I have done the required. Please review.

**Comment by dwertent** [comment]
@RayAsh37 will it update automatically when changing the code? I not, we should set it that way.

**Comment by RayAsh37** [comment]
@dwertent Maybe have to set it up, using GitHub actions instead. Looking into it

**Comment by dwertent** [comment]
We decided not to add codeSee since we do not want to constantly maintain the link.

**Pull request closed without merging**